src/test/regress/Makefile

```diff
@@ -111,6 +111,11 @@ check-base-mx: all
 	$(pg_regress_multi_check) --load-extension=citus \
 	-- $(MULTI_REGRESS_OPTS) --schedule=$(citus_abs_srcdir)/mx_base_schedule $(EXTRA_TESTS)
 
+check-minimal-mx: all
```

Comment:
Why do we need this schedule when we have `check-base-mx`?

Reply:
Same as what we have `check-minimal` for. `check-base-mx` adds some tables, which takes a while (~500ms) and is not needed for all tests. So if you don't need them for a new test you are creating can save some time when running it often.